MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -12,5 +13,13 @@ internal class MiniAppWebView: WKWebView {
         self.init(frame: .zero, configuration: config)
         contentMode = .scaleToFill
         load(urlRequest)
+        evaluateJavaScript("navigator.userAgent") { [weak self] (result, error) in
```

Comment:
Ah, too bad there's no way to access this from swift 😣  Android lets you retrieve it from the webview settings.

Reply:
Isn't this accessible?
https://developer.apple.com/documentation/webkit/wkwebview/1414950-customuseragent

Reply:
@ts-nitish-a-jain customUserAgent string will be empty in the first place. Check here https://developer.apple.com/forums/thread/106744